recipes/easy_profiler/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class EasyProfilerConan(ConanFile):
+    name = "easy_profiler"
+    description = "Lightweight profiler library for c++"
+    license = "MIT"
+    topics = ("conan", "easy_profiler")
+    homepage = "https://github.com/yse/easy_profiler/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
```

Comment:
```suggestion
    }
    short_path = True
```

it requires shot path to pass on Windows build.